client/src/components/Results/ResultsMap.js

```diff
@@ -22,7 +20,56 @@ const styles = {
   },
 };
 
-const useStyles = makeStyles((theme, props) => ({
+// the map layer that shows the bubbles
+const clusterLayer = {
+  id: "clusters",
+  type: "circle",
+  source: "stakeholders",
+  filter: ["has", "point_count"],
+  paint: {
+    "circle-color": [
+      "step",
+      ["get", "point_count"],
+      "#51bbd6",
```

Comment:
No hard-coded colors. It would be best if the theme file is the only source of truth for hex or rgb colors.